src/server/bitops_family.cc

```diff
@@ -1159,7 +1165,7 @@ void BitOp(CmdArgList args, ConnectionContext* cntx) {
   cntx->transaction->Execute(std::move(shard_bitop), false);  // we still have more work to do
   // All result from each shard
   const auto joined_results = CombineResultOp(result_set, op);
-  // Second phase - save to targe key if successful
+  // Second phase - save to target key if successful
```

Comment:
check the replication journal code line 1185. need to fix this also

Reply:
Wow. I blindly did not both with the journal because I `thought that we replicate the same command always`. But in this context, we do not. What we do instead is we take the result of the operation and call `SET` directly which is a `very nice` optimization since we don't now need to run the command on the replica but rather do an `update` on the result directly.

This is a nice pattern/optimization to know. +1